### PR TITLE
Add pystack recipe

### LINKED ISSUE #1602: Add pystack
https://github.com/eklitzke/pystack

### PR DESCRIPTION
**Pull request by gqmelo**
Fix #1602

### CODE REVIEW

File: recipes/pystack/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "pystack" %}
+{% set version = "0.0.0" %}
```
Comment (gqmelo) [inline recipes/pystack/meta.yaml]: There is no release, not sure what version should be

File: recipes/pystack/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set name = "pystack" %}
```
Comment (gqmelo) [inline recipes/pystack/meta.yaml]: There is a package on PyPI with the same name. Should I keep this name here?
I noted that when there is a conflict, the PyPI package is normally named ```python-*```

Comment (jakirkham) [inline recipes/pystack/meta.yaml]: Can we raise an issue and point out this naming conflict to them? Maybe they would be willing to rename given the confusion this is causing.

Comment (gqmelo) [inline recipes/pystack/meta.yaml]: OK, I created an issue to address that.

**Comment by gqmelo** [comment]
So apparently the project was deleted from github and running gdb in batch give almost the same result:
```gdb -p PID -batch -ex "py-bt"```

So I'm closing this

**Pull request closed without merging**